### PR TITLE
A simplify way of changing measurement units

### PR DESCRIPTION
**Pull request by nonsignificantp**
Currently, the haversine function accepts two parameters as a way of changing the default unit of measurement. Both parameters take a Boolean variable that is used later inside an if...else statement to set the preferable unit.

The way I see it, the current state give rise to two problems:

1. Both parameters can be set to True without rising any exception.
2. Adding new units results in more parameters and a longer if...else statement.

On this pull request I propose a unify way of changing the measurement unit that makes the package easier to use and opens the possibility to add multiple new units by just adding new key:values to a dictionary.

In this ways the function calling goes from this:

```Python
haversine(coord1, coord2, miles=True)
```

to this:

```Python
haversine(coord1, coord2, unit='mi')
```

Concurrently, a new set of units where added: meters (unit='m'), inches ('in') and feets ('ft').

Last but not least, I remove the function from the `__init__.py` file and now is inside `haversine.py`. 

I'm open to changes and discussion.

### CODE REVIEW

File: haversine/haversine.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    feets (ft) and inches (in).
+
+    """
+    # mean earth radius - https://en.wikipedia.org/wiki/Earth_radius#Mean_radius
+    AVG_EARTH_RADIUS_KM = 6371.0088
+
+    # Units values taken from http://www.unitconversion.org/unit_converter/length.html
+    conversions = {'km': 1,
+                   'm': 1000,
+                   'mi': 0.621369949,
```
Comment (jdeniau) [inline haversine/haversine.py]: The conversion ratio seems to be `0.621371192`

**Comment by jdeniau** [comment]
Just for the record, I will need to bump a new major tag, as it breaks compatibility for non-kilometer users.

**Comment by jdeniau** [comment]
Well don't bother, The package nearly never change, and the CHANGELOG will be very small. I don't think this is really needed

**Comment by jdeniau** [comment]
Pubished as `2.0.0`. https://pypi.org/project/haversine/2.0.0/

Thanks for you contribution !

**Comment by nonsignificantp** [comment]
You're welcome!

**Pull request merged**